src/Athens-Examples/AthensTigerWindow.class.st

```diff
@@ -4,6 +4,7 @@ I'm an example that shows a tiger that moves via rotation, translation and zoom
 Class {
 	#name : 'AthensTigerWindow',
 	#superclass : 'StandardWindow',
+	#classTraits : '{} + TraitedClass',
```

Comment:
Hmm, we should fix this, this is the default class trait

Reply:
(not in this PR)

Reply:
Do you have an idea where it comes from because I did not touch this part.

Reply:
Yes, it's the fix we did the other day to Traits

Reply:
Yes I was connecting after...:)